### DISCUSSION TITLE
Editor Lifecycle - Activating the Cell with a Single Click

### DISCUSSION THREAD

**Discussion by opachano** [Comcast/react-data-grid#2290] [Q&A] [answered]
Hello Everyone,

I am currently using a custom editor for cell editing and have as a requirement for users to be able to activate the cell to use the custom editor with just one click, is there a way to do this? I understand the current Editor Lifecycle initiates with either double click or enter, but that would not work in this scenario. Thank you in advance for any answer you provide!

**Comment by ekeuus** [comment]
I was also thinking that it could be interesting to provide a prop to the formatter such as onEditStart which oke could initiate whenever

**Comment by nstepien** [comment, marked as answer]
If you're on the latest canary version, I believe `column.editorOptions = { editOnClick: true };` should work.
